validphys2/src/validphys/filters.py

```diff
@@ -43,6 +44,41 @@
 }
 
 
+def _get_kinlabel_process_type(process_type):
+    """Get KIN_LABEL from the dictionary above according
+    to the process type
+    This requires some extra digestion for DIS
+    """
+    if process_type[:3] == "DIS":
+        return KIN_LABEL["DIS"]
+    return KIN_LABEL[process_type]
+
+
+# TODO: in the new commondata instead of having this, let's always use the same
+# variables
+def _variable_understanding(variables_raw, process_vars):
+    """Given a set of variable, check whether it might be a variation of existing
+    variables for a process type"""
+    variables = [i for i in variables_raw]
+
+    def substitute(pr_v, cd_x):
+        if pr_v in process_vars and cd_x in variables:
+            variables[variables.index(cd_x)] = pr_v
+
+    substitute("eta", "y")
+    substitute("eta", "eta")
+    substitute("etay", "eta")
+    substitute("etay", "y")
+    substitute("p_T2", "pT_sqr")
```

Comment:
In hindsight, this variable can be easily confused with `pT_sqrt`. I am not sure if is still easily possible to change this.
```suggestion
    substitute("p_T2", "pT_sqrd")
```

Reply:
The pT squared is now `pT2`

That list of substitutions is there only for debugging, that function will be removed at the end (once I've checked everything and can then substitute the variables at the top of the file).